daemon/cmd/config.go

```diff
@@ -156,16 +156,18 @@ func (h *getConfig) Handle(params GetConfigParams) middleware.Responder {
 
 	d := h.daemon
 	m := make(map[string]interface{})
+	option.Config.ConfigPatchMutex.RLock()
 	e := reflect.ValueOf(option.Config).Elem()
 
 	for i := 0; i < e.NumField(); i++ {
 		if e.Field(i).Kind() != reflect.Func {
 			// Remove configurable opttions from read-only map
-			if e.Type().Field(i).Name != "Opts" {
+			if e.Type().Field(i).Name != "Opts" || e.Type().Field(i).Name != "ConfigPatchMutex" {
```

Comment:
```suggestion
			if e.Type().Field(i).Name != "Opts" && e.Type().Field(i).Name != "ConfigPatchMutex" {
```

This should be an AND, since we want to skip both "Opts" and "ConfigPatchMutex".

Reply:
I am sorry. Silly mistake I'll just correct it.